judgments/templatetags/query_filters.py

```diff
@@ -68,3 +68,17 @@ def remove_court(query_params, court):
     params["page"] = None
     params["court"] = [court2 for court2 in params.get("court", []) if court != court2]
     return make_query_string(params)
+
+
+@register.filter
+def replace_year_in_query(query_params, year):
+    params = dict(query_params)
+    params["from"] = None
+    params["from_day"] = None
+    params["from_month"] = None
+    params["from_year"] = year
+    params["to"] = None
```

Comment:
Could we delete these parameters instead (if they exist) to make the parameters nicer?

Reply:
Probably not essential for POC.